extensions/map-layers-formats/src/GoogleMaps/GoogleMapDecorator.ts

```diff
@@ -0,0 +1,95 @@
+import { CanvasDecoration, DecorateContext, Decorator, IconSprites, IModelApp, Sprite } from "@itwin/core-frontend";
+import { Point3d } from "@itwin/core-geometry";
+import { MapTypes } from "./GoogleMaps";
+
+
+/** A simple decorator that show logo at the a given screen position.
+ * @internal
+ */
+export class LogoDecoration implements CanvasDecoration {
```

Comment:
Eventually we're going to have to deal with multiple logos trying to overlay on the same area of the viewport (e.g., the imagery provider and the terrain provider). It's fine for now.